app/models/manageiq/providers/openstack/cloud_manager/vm/associate_ip.rb

```diff
@@ -0,0 +1,47 @@
+module ManageIQ::Providers::Openstack::CloudManager::Vm::AssociateIp
+  extend ActiveSupport::Concern
+
+  included do
+    supports :associate_floating_ip do
+      if cloud_tenant.nil? || Rbac.filtered(cloud_tenant.floating_ips).empty?
```

Comment:
@mansam Yea, documentation is a bit light there.

@gtanzillo Do you have ideas on when someone would add their model to the classes that rbac supports?